CHANGES.md

```diff
@@ -1,6 +1,323 @@
 Release Changes List
 ====================
 
+TODO
+o missing contributions from interns (or others?)
+o any missing cross-references?
+o new compiler flags (and developer compiler flags)
+o examples directory
+o spellecheck
+o `` vs. ''
+o docs/1.28/...
+o linefeeds
+
+version 1.28.0
+==============
+
+Thirty-first public release of Chapel, September 15, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+
+Packaging / Configuration Changes
+---------------------------------
+* made `CHPL_MEM` default to `cstdlib` on Arm-based Macs
+
+Syntactic / Naming Changes
+--------------------------
+
+Semantic Changes / Changes to the Chapel Language
+-------------------------------------------------
+* updated `.low`/`.high` on strided ranges/domains to return aligned bounds  
+  (see https://chapel-lang.org/docs/1.28/language/spec/ranges.html#ChapelRange.range.low  
+   and https://chapel-lang.org/docs/1.28/language/spec/ranges.html#ChapelRange.range.high)
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+* marked comparison operations between types as unstable for the time being
+  (e.g., `type1 <= type2` is currently considered unstable)
+* made array methods `.count()`, `.find()`, `.reverse()`, `.sorted()` unstable
+* marked `locale.numPUs()` as being unstable for the time being
+* deprecated assignments/initializations of `shared` classes with `owned`
+* deprecated support for directly creating new borrowed class instances
+  (i.e., `new borrowed Class()` is now deprecated)
+* deprecated `locale.callStackSize()` with no intention to replace it
+* deprecated support for the `<~>` operator
+  (see https://chapel-lang.org/docs/1.28/language/spec/statements.html#the-i-o-statement)
+* deprecated the NUMA locale model and related methods on `locale` values
+* deprecated the `alignedBoundsByDefault` config param which has no effect
+  (see https://chapel-lang.org/docs/1.28/language/spec/ranges.html#ChapelRange.alignedBoundsByDefault)
+* removed the previously deprecated `arrayIndicesAlwaysLocal` config param
+* removed the previously deprecated `sizeReturnsInt` config param
+
+New Features
+------------
+* added a new `@unstable` annotation to indicate symbols that may evolve
+  (see TODO)
```

Comment:
Here's the link that should go in here: https://chapel-lang.org/docs/1.28/developer/bestPractices/Unstable.html#best-practices-unstable